api/sso/routes.py

```diff
@@ -52,8 +52,11 @@ def logout(self):
         azure_ad_sign_out_url = (
             Config.AZURE_AD_AUTHORITY
             + "/oauth2/v2.0/logout"
-            + "?post_logout_redirect_uri="
-            + post_logout_redirect_uri
+            + (
+                "?post_logout_redirect_uri=" + post_logout_redirect_uri
+                if len(post_logout_redirect_uri) != 0
```

Comment:
Is this checking the length of the string? If so we could just do a truthy:

```suggestion
                if post_logout_redirect_uri
```

Reply:
perfect!